study.md

```diff
@@ -39,7 +41,12 @@ sum = left_operand.to_i + right_operand.to_i
 Write some JavaScript code that duplicates this functionality.
 
 ```javascript
-// your answer here
```

Comment:
We want the two operands to start out as strings and then get converted to numbers before being added together.

```javascript
const leftOperand = '1';
const rightOperand = '2';

const sum = parseInt(leftOperand) + parseInt(rightOperand);
```